src/sync_connection_wrapper/mod.rs

```diff
@@ -73,6 +73,7 @@ fn from_tokio_join_error(join_error: JoinError) -> diesel::result::Error {
 /// #    some_async_fn().await;
 /// # }
 /// ```
+#[derive(Debug)]
 pub struct SyncConnectionWrapper<C> {
     inner: Arc<Mutex<C>>,
```

Comment:
Thanks for submitting this PR.
Can you explain why this would be actual helpful for your use-case as `#[derive(Debug)]` only implements `Debug` if all generic parameters (in this case `C`) implements `Debug` on their own. I think all of the possible types for `C` (especially the `SqliteConnection` type) do not implement `Debug` on their own, so this won't actually be used at all.

Reply:
It's a first step. I think need to work on e.g. `SqliteConnection`.

Reply:
As I had that discussion already a few times: The connection types do not implement `Debug` because there is nothing you can reasonably show there. They only hold a few opaque pointers to c objects. We also do not want to store the database URL in their as that could leak confidential information (login credentials, etc).

Reply:
Fair enough. I will close this PR then.